### DISCUSSION TITLE
Clean Dashboard Pop Out Window to show all Server or Workstations or Both

### DISCUSSION THREAD

**Discussion by dkleckner498** [amidaware/tacticalrmm#1033] [Ideas]
I was trying to find to see if a request has been added for this feature or not.  If so please let me know and we can close this one out.  But to do a pop out Clean Dashboard that shows either Workstations or Servers that you would display on a big screen to show troubled or non passed checked assets that need attention.

**Comment by dinger1986** [comment]
so it would only show machines with issues?

**Reply by dkleckner498** [reply]
Correct, maybe the client and site and the machine that has issues.  I mean thats one way of doing it.  There is another way, and I am going to refer to another piece of software with a dashboard where it shows all of the Servers or Workstations with a Green Icon for each but if something is wrong it either changes yellow or red for the icon of that asset.  I am fine either way just something that would be good to have to display.

**Comment by dinger1986** [comment]
you can view them all and it should reorder them when theres errors etc and show up green, blue, yellow or red depending on status. Maybe a mockup would be useful to show what you mean?

Theres a grafana 3rd party plugin if that helps (I scripted it up but other members of the community wrote it)? https://docs.tacticalrmm.com/3rdparty_grafana/

**Comment by dkleckner498** [comment]
I do have Grafana with that dashboard already setup which is good but looking for something even more simpler than that I guess.   I will create a markup document but something similar to the screenshot.  For purposes I removed what software that is coming from but something that shows devices and the client which in this case would be servers and then maybe a scrollable text on the bottom of alerts or that all alerts have been cleared.
![Screenshot 2022-03-30 121035](https://user-images.githubusercontent.com/98860482/160881479-7a815b3d-0ba4-4e77-b219-2f6cdf0ed5aa.png)

**Comment by silversword411** [comment]
...so your idea is if you had a NOC with 1/many TV's as HUD's for issues you'd have a scrolling pane of glass showing status of status: Good/bad, and what the issue is?

**Comment by dkleckner498** [comment]
That would be correct.

**Comment by dkleckner498** [comment]
Well and also alongside the scrolling, just something similar to that screenshot but could be a listing to help fit assets and to bring to the top client names that have assets in error.
